src/dune_config_file/feature_flags.ml

```diff
@@ -0,0 +1 @@
+let cache_enabled_by_default = false
```

Comment:
What is the point of "feature flags" when compared to the runtime configuration we already have? Is it that they are only set at compile time?